### DISCUSSION TITLE
Ioc dynamic call [v5]

### DISCUSSION THREAD

**Discussion by sasha172017** [adonisjs/core#1819] [Help] [answered]
How can i call dynamic services in 5v which register in AppProvide, for example 

const { ioc } = require('@adonisjs/fold')

  const Config = app.use('Adonis/Src/Config')

how can i do it?

**Comment by thetutlage** [comment, marked as answer]
You can access the container instance for the app as follows:

```ts
import Application from '@ioc:Adonis/Core/Application'
Application.container.use('Adonis/Src/Config')
```

**Reply by sasha172017** [reply]
thank you, again) but why is it  missing in documentation(
